source/fundamentals/data-formats/document-data-format-pojo.txt

```diff
@@ -18,19 +18,17 @@ MongoDB Java driver using plain old Java objects (`POJOs <https://en.wikipedia.o
 POJOs are often used for data encapsulation, separating business logic from
 data representation.
 
-In the following sections we show:
-
-- A POJO example
-- How to configure the driver to serialize and deserialize POJOs
-- Sample code to read and write documents to MongoDB using POJOs
+In the following sections we show an example on how to configure the
+driver to serialize and deserialize POJOs, and how to read and write to
+documents using POJOs. 
```

Comment:
Suggestion: since this is a list of items, it might be easier to identify as a bulleted list.